### PR TITLE
[EOC-457] Bug live demo does not work

### PR DESCRIPTION
**Pull request by adamklepacz**
Seeding data works as expected.
Demo account work as expected.

### CODE REVIEW

File: server/seed/demoSeed/generateItems.js [hunk 1]
```diff
@@ -9,31 +9,60 @@ const generateItems = (demoUserId, userIds) => [
     _id: ObjectId(),
     authorId: demoUserId,
     description: '',
+    editedBy: ObjectId(),
```
Comment (alemikolo) [inline server/seed/demoSeed/generateItems.js]: I'm not sure, but I think this shouldn't be fake id. These items will be displayed to the demo user and controller will try to populate this data from users collection. Replace with one of demo users id like in authorId's case eg. `[userIds[2]]`

File: server/seed/generateItems.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const generateItems = userId => itemsCount => {
+  const items = [];
+
+  for (let i = 0; i < itemsCount; i += 1) {
+    items.push({
+      _id: ObjectId(),
+      authorId: userId,
+      authorName: 'Adam',
+      description: '',
```
Comment (alemikolo) [inline server/seed/generateItems.js]: editedBy ?

File: server/seed/generateItems.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ const generateItems = userId => itemsCount => {
       authorId: userId,
       authorName: 'Adam',
       description: '',
+      editedBy: ObjectId(),
```
Comment (alemikolo) [inline server/seed/generateItems.js]: userIds[1] instead of ObjectId(),

File: server/seed/generateItems.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+const {
+  Types: { ObjectId }
+} = mongoose;
+
+const generateItems = userId => itemsCount => {
+  const items = [];
+
+  for (let i = 0; i < itemsCount; i += 1) {
```
Comment (marekrozmus) [inline server/seed/generateItems.js]: `i++`

Comment (adamklepacz) [reply server/seed/generateItems.js]: Zrzut ekranu 2019-09-9 o 11 34 19

Why` i++` ??
Till now we have used `i += 1` across the whole project

Comment (adamklepacz) [reply server/seed/generateItems.js]: BTW eslint disallows
Zrzut ekranu 2019-09-9 o 11 40 01
 usage of i++

Comment (marekrozmus) [reply server/seed/generateItems.js]: please check rule: { "allowForLoopAfterthoughts": true } 
https://eslint.org/docs/rules/no-plusplus

In loops there is no problem with using it

**Comment by adamklepacz** [comment]
> please check rule: { "allowForLoopAfterthoughts": true }
> https://eslint.org/docs/rules/no-plusplus
> 
> In loops there is no problem with using it

@marekrozmus 
I've added rule and ++, **but still curious why ++ instead of +=1 ??**

File: server/seed/generateItems.js (hunk 4 above)
Comment (Ciunkos) [inline server/seed/generateItems.js]: Consider this helper:

```javascript
const range = n =>
  Array(n)
    .fill()
    .map((_, i) => i);

const items = range(itemsCount).map(i => ({ _id: _id: ObjectId(), name: `item ${i}` }));
```

File: server/seed/demoSeed/generateCohorts.js [hunk 5]
```diff
@@ -10,6 +10,11 @@ const generateCohorts = (demoUserId, userIds) => [
     description:
       'You are the owner of this cohort. You can create sacks, add members and manage their permissions, modify and archive cohort.',
     isArchived: false,
+    isDeleted: false,
```
Comment (Ciunkos) [inline server/seed/demoSeed/generateCohorts.js]: Consider using constructors so default parameters can be assigned automatically:

```javascript
const Cohort = ({ description = '', isArchived = false, locks = { description = false, name = false } = {}, ... }) => ({
  _id: ObjectId(),
  description,
  isArchived,
  locks,
  ...
});

const generateCohorts = (demoUserId, userIds) => [
  Cohort({ description: 'You are the owner of this cohort. You can create sacks, add members and manage their permissions, modify and archive cohort.' }),
  ...
]
```

File: server/seed/seedDb.js [hunk 6]
```diff
@@ -33,6 +33,6 @@ const seedDatabase = async () => {
 };
 
 seedDatabase().catch(error => {
-  console.log('☠️  Something went terribly wrong:', error);
+  console.error('☠️  Something went terribly wrong:', error);
```
Comment (Ciunkos) [inline server/seed/seedDb.js]:
```suggestion
  console.error('☠️ Something went wrong:', error);
```

**Pull request merged**